sdk/eventhub/ci.yml

```diff
@@ -42,3 +42,7 @@ stages:
       safeName: AzureMessagingEventHubs
     - name: Azure.Messaging.EventHubs.Processor
       safeName: AzureMessagingEventHubsProcessor
+    - name: Microsoft.Azure.Management.EventHub
+      safeName: Microsoft.Azure.Management.EventHub
```

Comment:
Not this one. Actually these 3.

Microsoft.Azure.EventHubs
Microsoft.Azure.EventHubs.Processor
Microsoft.Azure.EventHubs.ServiceFabricProcessor

Reply:
Right, fixed. The pipeline still has an issue with these, but am looking into it